### DISCUSSION TITLE
Cant get zigbee2mqtt to start anymore

### DISCUSSION THREAD

**Discussion by 2proof4u** [Koenkk/zigbee2mqtt#13719] [Devices]
Hi

After the power went out to my PC, I can't get Zigbee2mqtt to start anymore. It usually works if I remove the cordinator from the usb and reinsert it again. I know the cordinator works because in desperation I tried to configure it with ZHA (deleted again)

 It just doesn't want to connect to Zigbee2mqtt anymore and just stands and uses 70% of cpu power and fills up the log every 2 sec. I am sure my ID for the usb cordinator is correct as I also tried to copy-paste it again just to try it.

Does anyone have a idee what the problem could be or what I should try.

debug 2022-08-27 22:49:36: Loaded state from file /config/zigbee2mqtt/state.json
info  2022-08-27 22:49:36: Logging to console and directory: '/config/zigbee2mqtt/log/2022-08-27.22-49-36' filename: log.txt
debug 2022-08-27 22:49:36: Removing old log directory '/config/zigbee2mqtt/log/2022-08-27.22-49-11'
info  2022-08-27 22:49:36: Starting Zigbee2MQTT version 1.27.0 (commit #unknown)
info  2022-08-27 22:49:36: Starting zigbee-herdsman (0.14.46)
debug 2022-08-27 22:49:36: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":false},"backupPath":"/config/zigbee2mqtt/coordinator_backup.json","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","databasePath":"/config/zigbee2mqtt/database.db","network":{"channelList":[25],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":65203},"serialPort":{"path":"/dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_e01fe0783286ec11837d641719c2d21c-if00-port0"}}'
error 2022-08-27 22:49:36: Error while starting zigbee-herdsman
error 2022-08-27 22:49:36: Failed to start zigbee
error 2022-08-27 22:49:36: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
error 2022-08-27 22:49:36: Exiting...
error 2022-08-27 22:49:36: SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Function.open (/app/node_modules/zigbee-herdsman/src/controller/database.ts:19:35)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:124:34)
    at Zigbee.start (/app/lib/zigbee.ts:58:47)
    at Controller.start (/app/lib/controller.ts:101:27)
    at start (/app/index.js:109:5)

**Comment by Koenkk** [comment]
Can you provide me the contents of your `data/database.db`?

**Reply by 2proof4u** [reply]
Hi

Hope you can help because the wife is not to happy:)

[database.zip](https://github.com/Koenkk/zigbee2mqtt/files/9439874/database.zip)

**Reply by Koenkk** [reply]
Can you replace it with this database.db and see if it works again?

[database.db.zip](https://github.com/Koenkk/zigbee2mqtt/files/9440123/database.db.zip)

**Reply by 2proof4u** [reply]
That did the trick.
Thanks alot

Did I do something wrong or can I do something to prevent something like this problem in the future?

**Comment by jerrm** [comment]
>  I tried to configure it with ZHA (deleted again)

Did you also disable ZHA?  HA can auto detect the stick and try to add ZHA back.

**Reply by 2proof4u** [reply]
I actually did not set it to ignore afterwards since it has been "ready" to configure ZHA for a long time.

anyway, I have now tried to set it to ignore but it is the same.
If there is another way to disable it, please advice me where and how to do it.

It might very well be that something has priority over the connector or preventing it to connect to Zigbee2mqtt first.
